atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/encoding/TwoPhaseEncodingStrategy.java

```diff
@@ -28,45 +28,37 @@
 import java.util.Arrays;
 import java.util.stream.Stream;
 
-public enum TwoPhaseEncodingStrategy implements TimestampEncodingStrategy<AtomicValue<TransactionStatus>> {
+public enum TwoPhaseEncodingStrategy {
     INSTANCE;
 
     private static final byte[] STAGING = new byte[] {0};
-    private static final byte[] COMMITTED = new byte[] {1};
+    static final byte[] COMMITTED = new byte[] {1};
```

Comment:
nit: The package visibility things should usually either be private, have visible for testing, or have a comment (unless I'm missing something really obvious)